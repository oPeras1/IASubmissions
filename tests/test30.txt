1	1	1	2	3	3	3	4	4	4	5	6	7	7	7	7	8	8	8	8
1	2	2	2	2	2	3	3	4	4	5	6	6	6	6	7	8	8	9	8
2	2	2	3	3	3	3	3	3	5	5	6	6	10	10	7	10	9	9	9
2	11	11	11	12	12	12	13	13	5	5	14	6	14	10	10	10	10	15	15
16	16	11	16	16	16	12	13	13	13	5	14	14	14	14	15	15	15	15	17
16	16	16	16	16	13	13	13	18	13	5	5	14	19	15	15	20	15	20	17
16	21	21	21	16	16	18	18	18	13	5	5	5	19	19	19	20	15	20	17
16	21	22	23	23	16	23	18	24	13	13	25	25	19	19	26	20	20	20	17
21	21	22	22	23	16	23	24	24	13	27	27	25	25	19	26	26	20	20	17
28	22	22	29	23	23	23	24	29	29	27	27	27	30	19	31	26	17	17	17
28	28	22	29	29	29	29	29	29	29	32	27	30	30	31	31	31	17	17	33
28	34	29	29	35	35	36	29	29	32	32	30	30	31	31	37	37	37	33	33
28	34	34	35	35	36	36	36	36	36	32	38	38	38	39	37	37	33	33	33
28	34	34	34	34	34	40	36	41	32	32	38	38	39	39	39	39	33	33	42
28	28	43	43	43	40	40	36	41	41	44	44	38	38	45	39	39	33	33	42
28	28	28	28	43	43	40	36	36	41	46	44	45	45	45	39	39	42	42	42
28	47	28	28	48	43	49	36	36	50	46	44	46	46	45	51	51	51	52	52
47	47	47	48	48	43	49	36	36	50	46	44	44	46	51	51	51	52	52	52
53	47	53	53	48	43	49	36	50	50	46	46	46	46	46	46	46	46	54	52
53	53	53	53	43	43	49	36	36	36	55	55	55	55	55	55	54	54	54	54