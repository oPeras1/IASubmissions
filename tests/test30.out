L	L	L	2	3	3	3	L	L	L	5	L	I	I	I	I	L	L	L	8
L	2	I	I	I	I	S	S	4	L	5	L	6	6	6	7	L	8	T	8
2	2	2	3	3	3	3	S	S	5	5	L	L	S	S	7	10	T	T	T
2	T	T	T	L	L	L	13	T	5	5	S	6	14	S	S	10	10	15	15
16	16	T	16	16	16	L	T	T	T	L	S	S	14	14	I	I	I	I	17
16	16	16	16	S	13	13	13	L	13	L	5	S	L	15	15	20	15	L	17
16	L	L	L	S	S	L	L	L	13	L	L	5	L	L	L	20	15	L	17
16	L	22	23	23	S	23	18	S	13	13	S	S	19	19	S	20	L	L	17
21	21	S	S	L	16	23	S	S	13	L	27	S	S	19	S	S	20	20	L
28	S	S	29	L	L	L	S	29	29	L	L	L	30	19	31	S	L	L	L
28	28	22	29	29	29	29	T	T	T	I	27	S	S	L	L	L	17	17	T
28	T	29	29	S	S	L	29	T	32	I	S	S	31	L	37	S	S	T	T
28	T	T	S	S	36	L	L	L	36	I	38	L	L	39	S	S	33	33	T
L	T	34	34	34	34	T	36	S	32	I	38	L	39	39	39	L	33	33	L
L	28	L	L	L	T	T	36	S	S	44	I	L	38	T	39	L	33	33	L
L	L	28	28	L	43	T	36	36	S	46	I	45	T	T	L	L	42	L	L
28	T	28	28	T	43	I	36	36	L	46	I	46	46	T	51	S	S	52	T
T	T	47	T	T	43	I	36	36	L	46	I	44	L	51	S	S	52	T	T
53	T	53	53	T	43	I	36	L	L	46	46	46	L	L	L	46	46	54	T
I	I	I	I	43	43	I	36	36	36	I	I	I	I	55	55	I	I	I	I